talentsearch/resources/js/mockTransform.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
added it as per David's suggestion and had moduleMapper in jest.config to use this .. But eventually I was able to run the tests without this. I think it can go now.